### PR TITLE
add <p> for error 404

### PR DESCRIPTION
**Pull request by nicolasnieloux**
Bonjour, voici un paragraphe concernant une erreur 404!!!! Et bonne année

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,5 +10,13 @@
...
     </header>
     <main></main>
+
+
+
+
+
+
+<p>Attention, ERREUR 404!!!!</p>
+
```
Comment (AnaisBonin) [inline index.html:20]: I would suggest to insert the paragraph into the main block :)
And remove the unecessary lines

`   <main>
<p>Attention, ERREUR 404!!!!</p>
</main>`

**Review by AnaisBonin — CHANGES_REQUESTED** [review]

**Review by baptiste-martin-campus — APPROVED** [review]
😱😱😱😱😱

**Comment by nicolasnieloux** [comment]
Solved PR!!!

**Comment by nicolasnieloux** [comment]
Fix PR!

**Review by CharlyLstudent — APPROVED** [review]
Bon boulot gamin

**Pull request merged**